### PR TITLE
Feature/added number validation

### PR DESCRIPTION
**Pull request by hoomberto**
# Description

Added function to check minutes and seconds on keypress - if a key is pressed that's not a digit, the corresponding error message is displayed

## Type of change

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Open index.html and try typing letters into the minutes and seconds fields

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -60,26 +60,16 @@ function checkLocalStorage() {
...
   var validCodes = [39, 37, 8, 9, 91, 13, 38, 40];
 
   if (event.keyCode >= 48 && event.keyCode <= 57 || validCodes.includes(event.keyCode)) {
+    hideError(2)
     hideError(3)
     return
   }
   event.preventDefault();
-  addError(3)
+  (event.target.id === "minutes") ? addError(2) : addError(3)
```
Comment (AshleyOh-bit) [inline main.js]: This ternary really cleaned up our functions. Very clever!

**Review by AshleyOh-bit — APPROVED** [review]

File: main.js (hunk 1 above)
Comment (tylrs) [reply main.js]: Thank you Ashley

File: main.js [hunk 2]
```diff
@@ -58,6 +60,18 @@ function checkLocalStorage() {
 }
 
 //form validation
+function checkDigits(event) {
+  var validCodes = [39, 37, 8, 9, 91, 13, 38, 40];
+
+  if (event.keyCode >= 48 && event.keyCode <= 57 || validCodes.includes(event.keyCode)) {
```
Comment (tylrs) [inline main.js]: So cool you figured out how to do this Ashley

File: main.js [hunk 3]
```diff
@@ -41,6 +41,8 @@ startActivityBtn.addEventListener("click", validateForm);
 iconSection.addEventListener("click", facilitateIconChange);
 logActivityButton.addEventListener("click", logActivity);
 createNewActivityButton.addEventListener("click", displayNewActivity);
+secondsInput.addEventListener("keydown", checkDigits);
+minutesInput.addEventListener("keydown", checkDigits);
```
Comment (tylrs) [inline main.js:45]: I didn't even know this was a type of event

**Review by tylrs — APPROVED** [review]
WE DID IT

File: main.js [hunk 4]
```diff
@@ -58,6 +60,18 @@ function checkLocalStorage() {
 }
 
 //form validation
+function checkDigits(event) {
+  var validCodes = [39, 37, 8, 9, 91, 13, 38, 40];
+
+  if (event.keyCode >= 48 && event.keyCode <= 57 || validCodes.includes(event.keyCode)) {
+    hideError(2)
+    hideError(3)
+    return
```
Comment (NicoleLatifi) [inline main.js:69]: The use of this return to break out of the function is really cool! I don't think the return on line 73 is necessary though.

File: main.js (hunk 2 above)
Comment (NicoleLatifi) [inline main.js]: For readability, I would add parentheses around the and conditional. So `(event.keyCode >= 48 && event.keyCode <= 57) || validCodes.includes(event.keyCode)`.

File: main.js [hunk 5]
```diff
@@ -58,6 +60,18 @@ function checkLocalStorage() {
 }
 
 //form validation
+function checkDigits(event) {
+  var validCodes = [39, 37, 8, 9, 91, 13, 38, 40];
```
Comment (NicoleLatifi) [inline main.js:64]: I wouldn't have thought to include these. Great job thinking about edge cases!

**Review by NicoleLatifi — APPROVED** [review]
Great job y'all! I also love how this PR is small and granular. Good job reusing the function for both the minutes and seconds too.

**Pull request merged**